src/main/java/leelawatcher/goboard/Position.java

```diff
@@ -164,6 +166,9 @@ public Position(Position prev, Move aMove) {
 
     if (aMove.isMove())      // now add the new stone
     {
+      if (!aMove.isPass()) {
```

Comment:
pass is a move, I think I'm going to tweak this and expose a static isPass utility method on Move.